.eslintrc.js

```diff
@@ -223,7 +222,7 @@ module.exports = {
       },
     },
   ],
-  parser: 'babel-eslint',
```

Comment:
can we get rid of babel-eslint then?

Reply:
not without peer dep warnings (https://github.com/facebook/fbjs/pull/405)